MergerService/WeatherForecast.cs

```diff
@@ -0,0 +1,13 @@
+namespace MergerService
```

Comment:
weather?

Reply:
that's the basic service template.
I kept it as its good example of service structure and should be removed when the service is added.